pages/linux/grub-script-check.md

```diff
@@ -0,0 +1,21 @@
+# grub-script-check
+
+> The program grub-script-check takes a GRUB script file and checks it for syntax errors.
+> It may take a path as a non-option argument. If none is supplied, it will read from standard input.
+> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dscript_002dcheck.html>.
+
+- Check a GRUB script file for syntax errors:
```

Comment:
```suggestion
- Check a specific script file for syntax errors:
```

There is no need to mention that script is for `GRUB` because the whole page is related to GRUB.